pages/osx/valet.md

```diff
@@ -0,0 +1,24 @@
+# valet
+
+> A minimalist Laravel development environment that allows hosting sites via local tunnels on http://<project>.test
+> More information: <https://laravel.com/docs/5.8/valet>.
+
+- Start the valet daemon:
+
+`valet start`
+
+- Register your current working directory as a path that Valet should search for sites:
+
+`valet park`
+
+- View your 'parked' paths:
```

Comment:
As the term 'parked' is not used in the previous example, I suggest using a similar description that way we could relate the examples
```suggestion
- View your registered paths on Valet:
```

Reply:
It is indeed mentioned in the previous example. `valet park`

Reply:
yes!